kafka/src/main/java/org/axonframework/extensions/kafka/eventhandling/consumer/subscribable/SubscribableKafkaMessageSource.java

```diff
@@ -211,6 +212,39 @@ public void close() {
         inProgress.set(false);
     }
 
+    private interface SubscriberBuilder {
```

Comment:
Nit: would you mind moving these into the `Builder` class? Although the addition makes sense, I find it a little obstructing otherwise.

Reply:
Now that I think about it, this isn't a solution that's specific to the `SubscribableKafkaMessageSource`. The `StreamableKafkaMessageSource` would also benefit from this.
So, I would either suggest moving the interface and classes to their own files, reachable by both classes, or dropping it entirely.

Reply:
Note that when upgrading these to their own files, JavaDoc and a copyright notice are requirements.

Reply:
I brought the `topics`, `addTopic` and `topicPattern` calls into an abstract KafkaSubscriberBuilder which both `SubscribableKafkaMessageSource.Builder` and `StreamableKafkaMessageSource.Builder` use. Please let me know your thoughts.